airflow/jobs/backfill_job.py

```diff
@@ -478,7 +478,7 @@ def _per_task_process(key, ti, session=None):
                         ti.set_state(State.SCHEDULED, session=session)
                 elif self.rerun_succeeded_tasks and ti.state == State.SUCCESS:
                     # Rerun succeeded tasks
-                    self.log.info("Task instance %s with state %s", ti, ti.state)
+                    self.log.info("Task instance %s with state %s, rerunning task", ti, ti.state)
```

Comment:
```suggestion
                    self.log.info("Task instance %s with state %s, rerunning task" succeeded, ti, ti.state)
```
Would that be better?